src/services/message-service.ts

```diff
@@ -21,6 +22,20 @@ export const createGetQueueMessage = (): ISocketMessage => {
   return createMessage({}, MESSAGE_TYPES.QUEUE_LIST);
```

Comment:
deconstruct MESSAGE_TYPES på starten. sånn at du kan skrive 

```suggestion
  return createMessage({}, QUEUE_LIST);
```

For å deconstrucvcte kan du skrive 

```
const {QUEUE_LIST} = MESSAGE_TYPES
```
osv.

Har fikset dette akkuratt her i min PR, men just in case du trenger å bruke sånt senere  :))